examples/fpga/artya7/util/vivado_hook_route_design_post.tcl

```diff
@@ -0,0 +1,27 @@
+open_project ../../lowrisc_ibex_top_artya7_0.1.xpr
+set saif_name "detailed_power.saif"
+
+regexp -- {Vivado v([0-9]{4})\.[0-9]} [version] -> year
+create_run impl_2 -parent_run synth_1 -flow "Vivado Implementation $year"
+set_property STEPS.OPT_DESIGN.IS_ENABLED false [get_runs impl_2]
+launch_runs impl_2
+wait_on_run impl_2
+
+open_run impl_2
+if {![file exists "$saif_name"]} {
+set_property top top_artya7 [current_fileset sim_1]
+launch_simulation -mode post-implementation -type functional
+open_saif "$saif_name"
+log_saif [get_objects -r *]
+add_force {/top_artya7/IO_CLK} -radix bin {1 0ns} {0 5000ps} -repeat_every 10000ps
```

Comment:
Is it necessary to specify the clock waveform in picoseconds?

Reply:
It was a copy paste from Vivado gui, that's why :) I fixed it now.

Reply:
Great!